src/components/SelectCoinModal.svelte

```diff
@@ -16,6 +16,17 @@
     dispatch('selected', { coin });
     active = false;
   };
+
+  // Show a loading message while we wait for tdex markets
+  // After a timeout delay, show a different message
+  const messages = {
+    loading: 'Loading...',
+    timeout: 'Something went wrong.',
+  }
+  let message = messages.loading;
+  const timeoutDelay = 60 * 1000; // 60 seconds
+  setTimeout(() => message = messages.timeout, timeoutDelay);
```

Comment:
this waiting time will happens all the time. How can we do better? (Maybe this includes to tweak the tdex-sdk that includes the gRPC/HTTP Client?)

Reply:
also, not sure is the right place, here in the select coin modal 🤔 

this provider discovery happen once we enter the app and most likely if the user see a message will reload again, making this waiting other 60 seconds